<launch>
    <arg name="fifo_path"  default="/tmp/opt/vm2m/var/run/intdash/logger_001.rx"/>
    <arg name="topic_name" default="/teleop/intdash_joy"/>
    <arg name="delay_threshold" default="1"/>
    
    <node pkg="teleop_reader_cpp" type="teleop_reader_cpp_node" name="teleop_reader_cpp_node" output="screen">
        <param name="fifo_path" value="$(arg fifo_path)"/>
        <param name="topic_name" value="$(arg topic_name)"/>
        <param name="delay_threshold" value="$(arg delay_threshold)"/>
    </node>
</launch>
